Backend/prisma/schema.prisma

```diff
@@ -12,7 +12,11 @@ datasource db {
 
 model users {
   id String @id @db.Uuid @default(uuid())
-  email String @db.VarChar(255)
+  email  String  @unique @db.VarChar(255)
   name  String @db.VarChar(255)
   phone String? @db.VarChar(255)
+  profilePicture String? @db.VarChar(255)
+  country String? @db.VarChar(255) 
+  university String?@db.VarChar(255)
+  language String? @db.VarChar(255)
```

Comment:
for phone maximum we need is 30. never need more than that
len("00963 (123) 1234 123456678") =26

longest country name is : 56
same for the others.

By the way, you don't need to overfocus on these, I can always take care of them in code review. just continue work as your are . Focusing on the most important thing (the logic)


```suggestion
  phone String? @db.VarChar(30)
  profilePicture String? @db.VarChar(255)
  country String? @db.VarChar(100) 
  university String?@db.VarChar(100)
  language String? @db.VarChar(100)
```

Reply:
I'll take care of those things next time, Thanks for the edit.